diff --git a/src/Codeception/TestCase/WPTestCase.php b/src/Codeception/TestCase/WPTestCase.php
index 8b28b82c6dd4bd3c4be67cfe23dfaa770ffe47e9..4d96c6ec656f51083ee5ec136047014c33e3ebf1 100644
--- a/src/Codeception/TestCase/WPTestCase.php
+++ b/src/Codeception/TestCase/WPTestCase.php
@@ -17,11 +17,6 @@ if (!class_exists('TracTickets')) {
     require_once dirname(dirname(dirname(__FILE__))) . '/includes/trac.php';
 }

-// Require the WordPress bootstrap file if not already loaded; this will deal with test methods running in isolation.
-if (!WPLoader::$didInit) {
-	require_once( __DIR__ . '/../../includes/bootstrap.php' );
-}
-
 // Load the PHPUnit compatibility layer.
 require_once __DIR__ . '/../../tad/WPBrowser/phpunit-compat.php';
 // phpcs:enable
